tests/components/orderentry/Paginate.test.jsx

```diff
@@ -10,13 +10,27 @@ const props = {
   fetchNew: jest.fn(),
 }
 
+const props2 = {
```

Comment:
@Rhotimee I would be better to remove this variable because you're not using it.

Reply:
I'm actually using it. There was an error before, but, it has been fixed.